codegen/golang.go

```diff
@@ -109,6 +109,9 @@ func NewTableFromStruct(name string, obj interface{}, opts ...TableOptions) (*Ta
 		column := ColumnDefinition{
 			Name: t.nameTransformer(field.Name),
 			Type: columnType,
+			Meta: ColumnMeta{
+				OriginalName: field.Name,
```

Comment:
Nit: Let's just put it as a field `OriginalName` ? this will eliminate another meta  `meta` definition ?